torch/csrc/jit/mobile/common_const.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+namespace torch {
+namespace jit {
+namespace mobile {
+
+constexpr auto kTensorJitIndex = "tensor_jit_index";
+
+} // namespace mobile
+} // namespace jit
+} // namespace torch
```

Comment:
Shouldn't need a separate constant files unless there are more than one. IMO we should try to avoid adding more files unless it helps with the structure of the codebase and understanding. If possible move it to one of the existing header files?

Reply:
I'm thinking to move the other constant string in export_module.cpp and import.cpp to this file, like "bytecode" and "constants" to avoid mistyping in next change. I haven't found a file under mobile folder to put constant variables, so I create this file, similar to [import_export_constants](https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/serialization/import_export_constants.h) from jit/serialization.